packages/module/sass/react-catalog-view-extension/_vertical-tabs.scss

```
@@ -5,28 +5,51 @@
 
   .vertical-tabs-pf {
     margin-block-end: 0;
+    padding-block-start: var(--pf-t--global--spacer--xs);
   }
 }
 
 .vertical-tabs-pf-tab {
   margin-block-start: var(--pf-t--global--spacer--sm);
+  margin-inline-start: var(--pf-t--global--spacer--md);
+  padding-inline-start: 0;
   position: relative;
 
   > a {
     color: var( --pf-t--global--text--color--regular);
     text-decoration: none;
     display: inline-block;
     font-size: 13px;
-    padding: var(--pf-t--global--spacer--sm) var(--pf-t--global--spacer--xs) var(--pf-t--global--spacer--sm) var(--pf-t--global--spacer--md);
+    margin-inline-start: var(--pf-t--global--spacer--md);
+    padding: var(--pf-t--global--spacer--xs) var(--pf-t--global--spacer--sm) var(--pf-t--global--spacer--xs) var(--pf-t--global--spacer--sm);
     vertical-align: top;
     width: 100%;
     word-break: break-word;
     border-radius: var(--pf-t--global--border--radius--small);
+    position: relative;
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      top: 0;
+      bottom: 0;
+      border-width: var(--pf-t--global--border--width--high-contrast--regular);
+      border-style: solid;
+      border-color: transparent;
```

Just a nit, we can use the action-plain border width token and change it for `:hover` below

```suggestion
      inset: 0;
      border: var(--pf-t--global--border--width--action--plain--default) solid var(--pf-t--global--border--color--high-contrast);
```